src/components/Course/CourseBaseTooltip.vue

```diff
@@ -1,11 +1,7 @@
 <template>
   <div class="course-tooltip">
     <div class="course-iconWrapper course-iconWrapper--caution">
-      <img
-        class="course-icon course-icon--caution"
-        src="@/assets/images/caution.svg"
-        alt="caution sign"
-      />
+      <img class="course-icon course-icon--caution" :src="icon" :alt="alt" />
```

Comment:
if you expand below, you will see that it's using the correct value. I defined the computed properties but forget to use it 😄